apps/core/components/passengerForm/DateInput.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+import React from 'react';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { format, setMonth } from 'date-fns';
+import { View } from 'react-native';
+import {
+  StyleSheet,
+  Picker,
+  Text,
+  TextInput,
+} from '@kiwicom/universal-components';
+
+type Props = {|
+  +onBirthDateSubmit: Date => void,
+|};
+
+type State = {|
+  +day: string,
+  +month: string,
+  +year: string,
+  +dayError: string,
+  +yearError: string,
+|};
+
+type Month = {|
+  value: string,
+  label: string,
+|};
+
+class DateInput extends React.Component<Props, State> {
+  state = {
+    day: '',
+    month: '',
+    year: '',
+    dayError: '',
+    yearError: 'string',
+  };
+
+  getMonthOptions = () => {
+    return [...Array(12).keys()].map<Month>(
+      (currentMonth: number): Month => ({
+        value: String(currentMonth),
+        label: format(setMonth(new Date(), currentMonth), 'MMMM'),
+      }),
+    );
+  };
+
+  handleMonthChange = (month: ?string) => {
```

Comment:
Is there a possibility to call handleMonthChange without a mount?

Reply:
I am not sure how to do that

Reply:
Sorry, probably it is misunderstood, I just don't understand why is the `mount` optional. ```(month: ?string)```

Reply:
Oh sorry.
It was because of `+onValueChange: (value: ?string) => void,` from `PickerTypes.js`
But honestly I dont understand why it was there. And as there might be reasoning behind it, I kept it like that

Reply:
Ah OK, thanks